modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -20,6 +132,41 @@ definition {
 		return ${curl};
 	}
 
+	macro assertCorrectObjectEntryValuesInResponse {
```

Comment:
what the `element` stands here for? 🤔

Reply:
The element is the josnPath in response, I mean the objectEntry jsonPath. It has been set to  "$.items[*].name" and "$..totalCount" in .testcase file. Would you like two methods assertCorrectObjectEntryNamesInResponse and assertCorrectObjectEntryCountsInResponse instead of the element?

Reply:
Updated element to objectJsonPath, is this ok?